lib/puppet_x/puppetlabs/powershell/powershell_manager.rb

```diff
@@ -101,8 +101,9 @@ def self.init_path
       end
 
       def make_ps_init_code(init_ready_event_name)
+        debug_output = Puppet::Util::Log.level == :debug ? '$True' : '$False'
         <<-CODE
-. #{self.class.init_path} -InitReadyEventName '#{init_ready_event_name}'
+. #{self.class.init_path} -InitReadyEventName '#{init_ready_event_name}' -EmitDebugOutput #{debug_output}
```

Comment:
@glennsarti I've given the user control over the output ... either via using `--debug` with Puppet or through `$DebugPreference`